queries/objdump/highlights.scm

```diff
@@ -0,0 +1,20 @@
+(address) @number
+(byte) @constant
+(hexadecimal) @number
+(integer) @number
+(section_address) @number
+
+(disassembly_section_label) @text.title
+[(file_offset) (discriminator)] @field
```

Comment:
```suggestion
[
  "file" "format"
  "File" "Offset:"
  "discriminator"
] @text

"Disassembly of section " @text.title
```

Reply:
Oh good idea, I hadn't thought of it like that